VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -659,6 +722,8 @@ export class ParticipantsPanelComponent implements OnInit, OnDestroy {
             participantToBeUpdated?.isLocalMicMuted(),
             participantToBeUpdated?.isLocalCameraOff()
         );
-        participant.assignPexipId(participantToBeUpdated?.pexipId);
+        if (participantToBeUpdated?.pexipId) {
```

Comment:
Is this check made here because we get a null reference exception for participantToBeUpdated? Probably best to check if participantToBeUpdated is not null instead of asserting the pexipId as this is already being evaluated in 726.

Reply:
Null check is done in line 726